articles/how-to-optimize-wordpress-website-speed/index.md

```diff
@@ -0,0 +1,214 @@
+the cached version will be served to subsequent users. Among the most
+popular WordPress caching plugins are [WP Rocket](https://wp-rocket.me/)
+(paid), [W3 Total Cache](https://wordpress.org/plugins/w3-total-cache/),
+[LiteSpeed Cache](https://wordpress.org/plugins/litespeed-cache/), [SG
+Optimizer](https://wordpress.org/plugins/sg-cachepress/), [WP Fastest
+Cache](https://wordpress.org/plugins/wp-fastest-cache/), and [WP Super
+Cache](https://wordpress.org/plugins/wp-super-cache/).
+
+### Optimize Images
+
+An image is worth a thousand words. You cannot avoid using images on
+your website. Images increase engagement and bring life to web
+content. In their original formats, image files are huge and could be
+more hurting than helping. The most common image formats used for the
+web are png, jpeg, and WebP. The png image format is uncompressed while jpeg is
+a compressed file format. The png is of higher quality thus larger file
+size. WebP combines the features of png and jpeg to let you create richer images that are smaller in file size than both png and jpeg. WebP offers 30% reduction in file size without compromising the quality. Just like png, WebP also provides transparency.
+#### WebP vs PNG
+WebP file size is 26% is smaller than png with the same quality.
+
+Due to the smaller file sizes, WebP loads faster than png.
+#### WebP vs JPEG
+WebP file size is 25% to 35% smaller with the same quality.
+
+Not all browsers can load WebP images. Some browsers will work with jpeg format.
+
+Reducing image size to reduce file is also necessary on
+top of choosing file formats. The main objective is to find a balance
+between file size and quality. Get the lowest file size and acceptable
+quality. Although there exists WordPress image compression plugins,
+compressing images before the upload is recommended. Photoshop,
+[compressor.io](https://compressor.io/) and [BulkResize](https://bulkresizephotos.com/en) are some of the tools used
+for image compression. Image compression plugins include [Imagify](https://imagify.io/wordpress/),
+[ShortPixel](https://wordpress.org/plugins/shortpixel-image-optimiser/),
+[Optimole](https://wordpress.org/plugins/optimole-wp/), and [WP
+Smush](https://wordpress.org/plugins/wp-smushit/). Keep your image under
+100 KB for the best performance.
+
+### Delete Unused Themes, Plugins, and Media Files
+
+This is a crucial website maintenance task. You should get rid of themes
+and plugins that are not in use. The plugins and themes take space in
+the server causing unnecessary strain. This involves deleting installed themes that are not in use and deleting deactivated plugins.
+
+### Use Well-optimized Themes
+
+Don't select some beautiful and impressive looking WordPress themes at
+the expense of performance. Pay special attention to theme optimization. Your
+WordPress theme should be elegant, lightweight, SEO friendly, fully
+responsive, and mobile optimized.
```

Comment:
Perhaps you could suggest some WP themes that are well-optimised and responsive like Astra and Page Builder Framework.